scripts/lint.sh

```diff
@@ -9,7 +9,8 @@ exitcode=0
 #  - readability/nolint: cpplint complains about excluded clang-tidy categories, which we widely use.
 #  - whitespace/braces: Does not work with {} initialization (see https://github.com/cpplint/cpplint/issues/204#issuecomment-1146769949).
 #  - build/include_subdir: Does not work well with our structure of using lib, test, and third_party directories.
-find src \( -iname "*.cpp" -o -iname "*.hpp" \) -print0 | parallel --null --no-notice -j 100% --nice 17 /usr/bin/env python3 ./third_party/cpplint/cpplint.py --verbose=0 --extensions=hpp,cpp --counting=detailed --filter=-legal/copyright,-runtime/references,-build/c++11,-readability/nolint,-whitespace/braces,-build/include_subdir --linelength=120 {} 2\>\&1 \| grep -v \'\^Done processing\' \| grep -v \'\^Total errors found: 0\' \; test \${PIPESTATUS[0]} -eq 0
+#  - readability/fn_size: We prefer longer methods if the data flow is continuous.
+find src \( -iname "*.cpp" -o -iname "*.hpp" \) -print0 | parallel --null --no-notice -j 100% --nice 17 /usr/bin/env python3 ./third_party/cpplint/cpplint.py --verbose=0 --extensions=hpp,cpp --counting=detailed --filter=-legal/copyright,-runtime/references,-build/c++11,-readability/nolint,-whitespace/braces,-build/include_subdir,-readability/fn_size --linelength=120 {} 2\>\&1 \| grep -v \'\^Done processing\' \| grep -v \'\^Total errors found: 0\' \; test \${PIPESTATUS[0]} -eq 0
```

Comment:
Not important, but have you checked with a 2024 version of cpplint if it still complains?

Reply:
Yes, stilld does not like 600 LoC methods.
```
src/benchmarklib/tpcds/tpcds_table_generator.cpp:1885:  Small and focused functions are preferred: TPCDSTableGenerator::_add_constraints() has 580 non-comment lines (error triggered by exceeding 250 lines).  [readability/fn_size] [1]
Category 'readability/fn_size' errors found: 1
Total errors found: 1
```

Reply:
And no, we cannot turn this off with `NOLINT`.